remo/sdk.py

```diff
@@ -394,7 +394,8 @@ def export_annotations(
         annotation_set_id: int,
         annotation_format: str = 'json',
         export_coordinates: str = 'pixel',
-        full_path: str = 'true',
+        full_path: bool = True,
```

Comment:
@SreeHarshaNelaturu if you to change `full_path` as well, you would need also convert bool to str in api.py, and check other places where `full_path` parameter uses